src/BlazorWebFormsComponents/HyperLinkField.razor

```diff
@@ -0,0 +1,10 @@
+﻿@typeparam ItemType
+@inherits BaseColumn<ItemType>
+
+@code
+{
+	RenderFragment RenderAnchor(string text, string url, string target)
+	{
+		return@<text><a href="@url" target="@target">@text</a></text>;
```

Comment:
I'm not sold on using the anchor component here because routing was not used in the original web forms controls.  In the future, when we enhance these components, we could consider it then as an alternate rendering, but for compatibility right now, this works great.

Reply:
Yeah I could see it either way. I think there are several places where we'll have to think about how the complex components are composed. For now I'll leave this as-is but I'll keep it in the back of my mind as we get to more complex scenarios.